src/data/prepopulated.ts

```diff
@@ -182,6 +182,19 @@ export const prepopulated: {
     voice: 'normal',
   },
 
+  'Voting in-person': {
+    riskProfile: 'average',
+    interaction: 'oneTime',
+    personCount: 2,
+
+    setting: 'indoor',
+    distance: 'sixFt',
+    duration: 10,
```

Comment:
Hmmm. I guess it depends on what kind of voting we're talking about. A ballot drop-off or filling out a whole ballot. What had you think 10 minutes?

Reply:
I think 10min is a reasonable estimate for how long it takes to sign the registry, get your ballot, and fill it out. Plus whatever part of the line is indoors, if there's a line.

Reply:
I just copied what Josh said in the bug. Do you think the numbers should be different? I haven't voted in person in a loooong time.

I'd be happy to add a mail-in dropoff scenario as well.

Reply:
I don't know if I've ever voted in-person, so I'm not a reliable source, either. Let's keep it at 10, then!

Seems like a ballot drop-off wouldn't be all that useful, given that this is already so few points.

Reply:
I think mail-in dropoff is basically free.